index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio setup mobile</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="container">
+    <header class="toolbar">
+      <a href="#" class="logo">My logo</a>
+      <nav class="navbar">
+        <div class="hambergur-button">
+          <img src="images/Icon - Menu.png" alt="hambergur-icon">
+        </div>
+      </nav>
+    </header>
+    <main class="main-content">
+      <div class="text-content">
+        <h1 class="heading-1">
+          I'm Lugard
+          <br>
+          Glad to see
+          <br>
+          you!
+        </h1>
+        <p class="paragraph">
+          I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2 class="paragraph-2">LET'S CONNECT!</h2>
```

Comment:
- [ OPTIONAL ] Great job on working on the design but I think here the heading for the h1 tag and for the h2 tag need to have more font weight to make your portfolio similar to the Figma design


- Your design 
![image](https://user-images.githubusercontent.com/58553711/192110243-6aee9d7d-bec1-4d1d-8320-ce3e79ce906f.png)

 - Figma design
 
![image](https://user-images.githubusercontent.com/58553711/192110263-b5d52873-a30a-45cc-a89a-44fb28fba70a.png)